dace/codegen/targets/cuda.py

```diff
@@ -1128,26 +1128,42 @@ def _emit_copy(self, state_id, src_node, src_storage, dst_node, dst_storage, dst
                     accum = '::template Accum%s' % reduction_tmpl
 
                 if any(symbolic.issymbolic(s, sdfg.constants) for s in copy_shape):
-                    callsite_stream.write(('    {func}Dynamic<{type}, {bdims}, {is_async}>{accum}({args});').format(
+                    # reduction not yet supported by template for dynamic case
+                    if accum or custom_reduction:
+                        raise NotImplementedError(f'reduction not supported by template {funcname}')
```

Comment:
Why is that the case? The C++ code allows it as far as I understand. Is it just a missing function?